### PR TITLE
Add glomap

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Needs colmap 3.11

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-c-cpp this is ready for review. Many thanks :)

File: recipes/glomap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - cmake
+    - make                           # [unix]
+    - pkg-config
+    - ninja
+    - llvm-openmp                    # [osx]
+    - libgomp                        # [linux]
```
Comment (xhochy) [inline recipes/glomap/meta.yaml:34]: Please also add these two to host, too.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/glomap/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipes/glomap/meta.yaml:34]: Thanks for the review @xhochy!

Can I ask if this has always been the case? In the [knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base/#openmp), it only mentions adding them to `build` and there are several recipes in which this is the case; e.g. in colmap which is now leading to version conflicts.

**Comment by Tobias-Fischer** [comment]
Thanks again @xhochy - this should be ready now. Note that there is a pin to libboost_devel 1.86 needed as the proper openmp pin is only available in a colmap version built for boost 1.86; I suggest that I can deal with that in the feedstock once it exists (simply remove the pin once the boost migration has been applied).

File: recipes/glomap/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipes/glomap/meta.yaml:34]: The issue is described at https://github.com/conda-forge/openmp-feedstock/issues/126 If it leads to issues here, it means that in one of your dependencies, you also should put it into host.

**Review by xhochy — APPROVED** [review]

**Comment by Tobias-Fischer** [comment]
Thanks @xhochy!

**Pull request merged**